packages/vuetify/src/components/VBottomSheet/VBottomSheet.ts

```diff
@@ -17,10 +18,10 @@ export default {
       default: 'auto'
     },
     persistent: Boolean,
-    value: null
+    value: Boolean
```

Comment:
I can't find the issue but there was a fair bit of fuss at one point about being able to bind any type to this.